### ISSUE TITLE
Fix global distribution

### ISSUE THREAD

**Issue by mrjones2014** [AndcultureCode/AndcultureCode.JavaScript.Quill#4]
The global distribution doesn't seem to be handling external peer dependencies correctly, and crashes when you try to import the package via a `<script>` tag in an HTML page. 

You can debug this using `./test-global-distribution.html`

**Comment by KevinBusch** [comment]
Took a quick peek.  Did you set up the webpack.config.js to be different than what's in `AndcultureCode.JavaScript.Core`?  See https://github.com/AndcultureCode/AndcultureCode.JavaScript.Core/blob/main/webpack.config.js.  Might explain why the peer dependencies aren't working...

**Comment by mrjones2014** [comment]
@KevinBusch I think it's slightly different but heavily based on the one you linked there.

**Comment by KevinBusch** [comment]
Interesting, yea I'm not sure what's going on here.  Are you sure the names are correctly labeled in `dependency-externals.json`?  I've seen where even casing in those names must be perfectly aligned with what the external dependency requires.

**Comment by mrjones2014** [comment]
Yeah, I tried lots of different things, but no matter what I did it wasn't working. It seems like what was happening was the peer dependencies were not being bundled with the library (which is the correct behavior) but then at runtime it would still try to `__WEBPACK_IMPORT__("peer-dep-name")` (which is the compiled method to get bundled dependencies) instead of looking for it in the external context.
